### PR TITLE
WRP-7677:  Updated babel-loader dependency to v9

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Babel-loader needs to be synchronized with the version enact cli is using

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated babel-loader dependency to v9.
There were no breaking changes affecting our code https://github.com/babel/babel-loader/releases 

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Minimum required node version is v14.15.0  https://github.com/babel/babel-loader/releases/tag/v9.0.0

### Links
[//]: # (Related issues, references)
WRP-7677

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian (daniel.stoian@lgepartner.com)

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by andreitirlalge — APPROVED** [review]
LGTM

**Comment by hong6316** [comment]
1. Write the node version required by the babel-loader in package.json
`"engines" : {
"node" : { ....`
2. Write the node version changes to the Additional Considers in the PR description
(Add a link to changing nodes in the babel-loader.)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Updated `babel-loader` version to `^9.1.2`.
```
Comment (hong6316) [inline CHANGELOG.md:3]:
```suggestion
* Updated `babel-loader` version to `^9.1.2`.
* Updated the minimum version of Node to `14.15.0`.
```

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**